### DISCUSSION TITLE
Tunnel Cloudflare

### DISCUSSION THREAD

**Discussion by avvalex** [indomus/forum#2454] [GENERALE - Discussioni generiche]
Buongiorno, chiedo un aiutino...
Ho un tunnel attivo e funzionante, e ora non mi dispiacerebbe usare lo stesso sistema anche per altre cose. Non capisco però questo:
- se volessi esporre, che so, l'interfaccia del router, devo fare un secondo tunnel con un'altra istanza dicker di cloudflared, o posso usare sempre quella?
- qualora volessi raggiungere la VPN del Fritz (ora che ho una linea nattata)  posso usare lo stesso sistema?
- qualora avessi qualche servizio che non richiede autenticazione, tipo Frigate, posso aggiungerla al tunnel apposito con qualche funzione di Cloudflare?
Chiedo perché la strada veloce sarebbe farsi dare il pubblico, però l'idea di avere IP privato+Cloudflared mi fa pensare a una maggior sicurezza.... O no?

**Comment by snakuzzo** [comment]
- Puoi usare un solo tunnel per N applicazioni. Maggiori info [QUI](https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/install-and-setup/tunnel-guide/local/local-management/ingress/)
- In teoria potresti farlo, ma porte e protocolli da utilizzare sono limitati. Maggiori info [QUI](https://developers.cloudflare.com/fundamentals/get-started/reference/network-ports/)
- Come scritto sopra, puoi aggiungere n "applicazioni" (così si chiamano, non servizi) e l'autenticazione per i singoli servizi devi gestirtela per conto tuo, magari con un reverse proxy, ma andresti solo a complicare l'architettura. Inoltre...a cosa dovrebbe servirti esporre quei servizi su internet? E' una cosa, a mio avviso, fuori di testa.

Per quanto riguarda l'ultima domanda...la sicurezza non dipende dall'avere IP pubblico o privato. Ed il discorso di cloudflared è totalmente indipendente dal tipo di IP che ti dà il provider

**Comment by avvalex** [comment]
Ho risolto facendomi attivare IP pubblico dinamico, così uso la buona vecchia VPN per connettermi alla rete domestica al bisogno (senza aprire alcuna porta o esporre alcun servizio)...
Grazie cmq per le delucidazioni (il tunnel che uso per HA funziona benissimo, per cui in caso mi servisse per altro ora so come muovermi). Ciao!
